SPECS/future/future.spec

```diff
@@ -38,42 +33,41 @@ BuildRequires:  python%{python3_other_pkgversion}-devel
 
 %if 0%{?with_python2}
 %package -n python2-%{name}
-Summary: Easy, clean, reliable Python 2/3 compatibility
 %{?python_provide:%python_provide python2-%{name}}
+Summary:        Easy, clean, reliable Python 2/3 compatibility
+BuildRequires:  python2-devel
+BuildRequires:  python2-numpy
+BuildRequires:  python2-pytest
+BuildRequires:  python2-requests
+BuildRequires:  python2-setuptools
+Provides:       future = 0:%{version}-%{release}
 %if 0%{?el6}
-BuildRequires: python-argparse, python-unittest2, python-importlib 
-Requires:      python-importlib
-Requires:      python-argparse
+BuildRequires:  python-argparse
+BuildRequires:  python-importlib
+BuildRequires:  python-unittest2
+Requires:       python-argparse
+Requires:       python-importlib
 %endif
 
-BuildRequires: python2-devel
-BuildRequires: python2-setuptools
-BuildRequires: python2-numpy
-BuildRequires: python2-requests
-BuildRequires: python2-pytest
-Provides:      future = 0:%{version}-%{release}
-
 %description -n python2-%{name}
 %{_description}
 %endif
 
 %if 0%{?with_python3}
```

Comment:
I think we can assume `with_python3` is always on.

Reply:
Closing the PR because this package doesn't need to be moved to core.